climbtrackclient/src/components/organisms/LogonForm/LogonForm.js

```diff
@@ -31,28 +30,18 @@ const LogonForm = () => {
     }, [authentication]);
 
     const [credentials, setCredentials] = useState({
-        emailAddress: '',
-        password: ''
+        emailAddress: 'climber@climber.com',
+        password: 'climbing'
     });
-    const onEmailTextFieldChange = (event) => setCredentials({ ...credentials, emailAddress: event.target.value });
-    const onPasswordTextFieldChange = (event) => setCredentials({ ...credentials, password: event.target.value });
-
-    let location = useLocation();
-    let { from } = location.state || { from: { pathname: "/" } };
+    const onEmailTextFieldChange = (event) => null  // setCredentials({ ...credentials, emailAddress: event.target.value });
+    const onPasswordTextFieldChange = (event) => null // setCredentials({ ...credentials, password: event.target.value });
 
-    const onLogonButtonClick = async () => {
-        const response = await axios.post('/api/login', credentials);
-        if (response.status == 200) {
-            setAccessTokens(response.data.accessToken, response.data.refreshToken);
-            setAuthenticated(true);
-            history.replace(from);
-            return;
-        }
-    };
+    const onLogonButtonClick = useCallback(() => dispatch(login(credentials)), [dispatch, credentials]);
```

Comment:
useCallback - to memoize this function?
Why are we watching dispatch - will this change?